charts/argo-events/Chart.yaml

```
@@ -16,3 +16,4 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - "[Added]: Ability to specify port for webhook deployment"
+    - "[Fixed]: Missing NATS version in values.yaml"
```

Looking at other pull requests and [docs here](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog), it seems you need to always provide just the latest change.

```suggestion
    - "[Fixed]: Missing NATS version in values.yaml"
```

---

@egze Thanks for the fix to the linter in `argo-events/README.md`.
About the suggestion you made about `argo-events/Chart.yaml`, I accepted your suggestion and used `Sign off and commit` right here in the PR UI, but now the DCO check fails. I thought the `Sign off and commit` would satisfy DCO requirements, but apparently it doesn't. So what do I need to do now? Re-commit locally with `git -s -m "..."` and re-push?

---

Yes, you can try it.